tests/panels/test_sql.py

```diff
@@ -225,14 +225,33 @@ def test_json_param_conversion(self):
         connection.vendor == "postgresql", "Test valid only on PostgreSQL"
     )
     def test_tuple_param_conversion(self):
+        """
+        For psycopg3 You cannot use IN %s with a tuple
+        https://www.psycopg.org/psycopg3/docs/basic/from_pg2.html#you-cannot-use-in-s-with-a-tuple
+        """
+
+        is_psycopg3 = True
+        try:
+            from psycopg import sql
+        except ImportError:
+            is_psycopg3 = False
+
         self.assertEqual(len(self.panel._queries), 0)
 
-        list(
-            PostgresJSON.objects.raw(
-                "SELECT * FROM tests_postgresjson WHERE field ->> 'key' IN %s",
-                [("a", "b'")],
+        if is_psycopg3:
+            list(
+                PostgresJSON.objects.raw(
+                    "SELECT * FROM tests_postgresjson WHERE field ->> 'key' = ANY(%s)",
+                    [["a", "b'"]],
+                )
+            )
+        else:
+            list(
+                PostgresJSON.objects.raw(
+                    "SELECT * FROM tests_postgresjson WHERE field ->> 'key' IN %s",
+                    [("a", "b'")],
+                )
```

Comment:
Since this test is only run on PostgreSQL anyway I think you could keep just the psycopg3 branch since it should work with psycopg2 as well.

Reply:
kept the psycopg3 version